src/Npgsql/NpgsqlCommand.cs

```diff
@@ -399,7 +399,7 @@ ELSE pg_proc.proargmodes
 END AS proargmodes
 FROM pg_proc
 LEFT JOIN pg_type ON pg_proc.prorettype = pg_type.oid
-LEFT JOIN pg_attribute ON pg_type.typrelid = pg_attribute.attrelid AND pg_attribute.attnum >= 1
+LEFT JOIN pg_attribute ON pg_type.typrelid = pg_attribute.attrelid AND pg_attribute.attnum >= 1 AND pg_attribute.attisdropped = FALSE
```

Comment:
This is a complete nitpick, but doesn't it look nicer to simply say `AND NOT pg_attribute.attisdropped`?

Reply:
> This is a complete nitpick, but doesn't it look nicer to simply say AND NOT pg_attribute.attisdropped?

You're probably right in both aspects :stuck_out_tongue_winking_eye: